src/main/java/frc/robot/subsystems/swerve/SwerveConstants.java

```diff
@@ -20,12 +23,16 @@ public record SwerveConstants(
 ) {
 
     public SwerveConstants(
+            String swerveName,
             double velocityAt12VoltsMetersPerSecond,
             Rotation2d maxRotationalVelocityPerSecond,
             PIDConstants translationMetersPIDConstants,
             PIDConstants rotationDegreesPIDConstants
     ){
         this(
+                "Subsystems/" + swerveName + "Swerve/",
+                "Subsystems/" + swerveName + "Swerve/State/",
+                "Subsystems/" + swerveName + "Swerve/Velocity/",
```

Comment:
kinda ugly, return the getSwerveLogPath that gets name, or give the fully logPath to constructor